src/lib/field/_selector.scss

```diff
@@ -545,11 +546,27 @@
         @include base.icon-color(invalid);
       }
     }
-    &--disabled {
+    &--disabled:not(.forge-field__addon-end-container--enabled) {
       ::slotted([slot='addon-end']) {
         @include base.icon-color(disabled);
       }
     }
   }
+
+  // Addon End Slot Always Enabled.
+  &.forge-field__addon-end-container--enabled.forge-field--disabled:not(.forge-field--invalid) {
+    .forge-field__addon-end-container {
+      @include theme.property(background-color, surface);
+      @include theme.property(color, text-primary-on-background);
+      @include theme.property(border-color, border-color);
+      border-style: solid;
+      border-width: 1px;
+      border-top-right-radius: 4px;
+      border-bottom-right-radius: 4px;
+      box-sizing: border-box;
+      -moz-box-sizing: border-box;
+      -webkit-box-sizing: border-box;
```

Comment:
Just an FYI, we run CSS through autoprefixer and these will be added automatically.

Reply:
Good point, this is a relic of some in browser testing, so we should removed this @Alex-Oxthorn